#!/usr/bin/env python2
PACKAGE = "acoustic_nav"

import roslib;roslib.load_manifest(PACKAGE)
from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("isStart",         bool_t,   0, "isStart parameter",      False)
gen.add("isAbort",         bool_t,   0, "to Abort the run",       False)
gen.add("search_depth",    double_t, 0, "search_depth",           1, 0, 10)
gen.add("search_distance", double_t, 0, "search_distance",        3, 0, 15)
gen.add("z_threshold",     double_t, 0, "z_threshold",            0.7, 0, 1)
gen.add("altitude_at_pinger",double_t, 0, "to find distance to pinger", 3.75, 0, 10)

exit(gen.generate(PACKAGE, "acoustic_nav", "acoustic_nav"))
